Tests/CommandsTests/BuildToolTests.swift

```diff
@@ -183,4 +183,24 @@ final class BuildToolTests: XCTestCase {
             }
         }
     }
+
+    func testParseableInterfaces() {
+        fixture(name: "Miscellaneous/ParseableInterfaces") { path in
+            do {
+                let result = try build(["--enable-parseable-module-interfaces"], packagePath: path)
+                XCTAssert(result.binContents.contains("A.swiftinterface"))
+                XCTAssert(result.binContents.contains("B.swiftinterface"))
+            } catch SwiftPMProductError.executionFailure(_, _, let stderr) {
+                XCTFail(stderr)
+            }
+
+            do {
+                let result = try build([], packagePath: path)
```

Comment:
Can you remove this check (or move it to some other existing test) since it is not very valuable and tests that perform a full build are expensive to run.